### PR TITLE
Converts homepage to mantine

### PR DESCRIPTION
**Pull request by jackgarritano**

### CODE REVIEW

**Comment by jackgarritano** [comment]
Unfortunately had to remove the 10% margin from the event component because I could find no other way to make the styling on the homepage look good. I tried using !important to overrule the 10% but the margins were still displayed. Hopefully this doesn't mess up other pages too much.

**Comment by jackgarritano** [comment]
Also had to remove the content div from the navbar component because it contained padding which prevented background color from covering entire area

File: src/pages/Home.tsx [hunk 1]
```diff
@@ -41,26 +35,30 @@ const Home: React.FC = () => {
   }, [currentUser]);
 
   return (
-    <div className="pageContainer">
+    // <div className="pageContainer">
```
Comment (samshefnie) [inline src/pages/Home.tsx:38]: remove comment if unnecessary

**Review by samshefnie — COMMENTED** [review]
looks great!

**Pull request merged**